sui_types/src/crypto.rs

```diff
@@ -183,6 +186,14 @@ impl std::fmt::Debug for PublicKeyBytes {
     }
 }
 
+pub fn random_key_pairs(num: usize) -> Vec<KeyPair> {
+    let mut key_pairs = Vec::with_capacity(num);
+    for _ in 0..num {
+        key_pairs.push(get_key_pair().1);
+    }
+    key_pairs
+}
```

Comment:
Nit: 
```rust
    let mut items = num;
    let mut rng = OsRng;
    std::iter::from_fn(if items == 0 {
        None
    } else {
        item -= 1;
        Some(get_key_pair_from_rng(&mut rng))
    })
    .collect::<Vec<_>>()
```
The `from_fn` is cosmetic, the reuse of the `Rng` is a (small) perf opt.